xbmc/video/windows/GUIWindowVideoBase.cpp

```diff
@@ -1497,7 +1497,7 @@ void CGUIWindowVideoBase::LoadPlayList(const std::string& strPlayList, int iPlay
     // load it
     if (!pPlayList->Load(strPlayList))
     {
-      CGUIDialogOK::ShowAndGetInput(6, 0, 477, 0);
+      CGUIDialogOK::ShowAndGetInput(6, 477);
```

Comment:
@da-anda 
Can you check out string ID 6?
This one is really weird and i've never really seen it actually in XBMC/Kodi itself
https://github.com/xbmc/xbmc/blob/master/addons/resource.language.en_gb/resources/strings.po#L53